packages/interpreter/src/ts/native.ts

```diff
@@ -238,6 +244,38 @@ export class NativeInterpreter extends JSChannel_ {
     }
   }
 
+  handleResizeEvent(entries: ResizeObserverEntry[], _observer: ResizeObserver) {
+    for (const entry of entries) {
+      const target = entry.target;
+      const realId = getTargetId(target)!;
+
+      const details = new ResizeEventDetail(
+        entry.borderBoxSize?.[0],
+        entry.contentBoxSize?.[0],
+        entry.contentRect,
+        entry.target
+      );
+
+      const event = new CustomEvent<ResizeEventDetail>("resize", { detail: details });
+      const contents = serializeEvent(event, target);
+
+      const body = {
+        name: "resize",
+        data: contents,
+        element: realId,
+        bubbles: false,
+      };
+
+      if (this.liveview) {
+        // TODO:
```

Comment:
Liveview defines the same IPC object so we can probably merge the two branches

Reply:
Ok.